core/example/benchmark/CMakeLists.txt

```diff
@@ -3,5 +3,7 @@ if(Cabana_ENABLE_Serial)
   target_link_libraries(NeighborListMDPerfTest cabanacore)
 endif()
 
-add_executable(PeakFlops Cabana_peakflops.cpp)
-target_link_libraries(PeakFlops cabanacore)
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "(x86)|(X86)|(amd64)|(AMD64)")
```

Comment:
Why are patterns between parentheses?
Why do you provide uppercase variants?

Reply:
Good question - I am unfamiliar with the syntax here.

Reply:
Just copy-paste from stackoverflow!

Reply:
You should have added a link to it for reference